README.md

```diff
@@ -1,8 +1,9 @@
 # Foosball
 
-![logo](/logo.png?raw=true "Project logo")
+![logo](/foosball/static/images/logo.png?raw=true "Project logo")
 
-Office foosball results!
+
+Office foosball results! Made with Python 3.5.
 
 LICENSE: MIT
```

Comment:
Btw, I'd like a mention where the logo is from as the resource is, while not shady and says "free for all use", it also is a bit vague on where it gets its own sources ;)